_data/sites.json

```
@@ -7182,6 +7282,21 @@
         ]
     },
 
+    {
+        "name": "Tutanota",
+        "url": "http://mail.tutanota.com/settings/subscription",
+        "difficulty": "hard",
```

```suggestion
        "difficulty": "easy",
```
This looks to be easy, according to the description you provided, isn't it?

---

Yes, it's easy. I copied the json template from some other account and forgot to change the difficulty when I added the Tutanota account deletion information.